media-collections/add-and-remove-assets-from-media-collections.md

```diff
@@ -0,0 +1,38 @@
+# Add and remove assets from media collection
+
+You can simply add or remove files and folders from media collections. Adding an asset to a media collection places it in a virtual group without redundancy while keeping the actual asset unaffected by any change in location or URL. Removing the asset simply removes the above association with the asset. Using this workflow makes it easy for an administrator to manage and share resources with users.
+
+Learn more about permissions for these operations for restricted users [here](../access-control-and-permissions/README.md#permissions).
+
+## Add assets to media collections
+
+In the ImageKit media library, select and right-click the files and folders you want to add to media collections and click on the "Add to media collections" option. You can only add assets if you have at least "contribute" permission to a media collection. A popup window would appear, allowing you to search for and select the media collections to which you wanted to add the assets. Only the media collections you can "manage" will appear in the list. After selecting the media collections, click on "Submit." 
+
+You can also create a new media collection through the option given in this popup.
+
+If you add a folder and its subfolder or file to a media collection, both will be added. You can also set separate permission levels for them.
+
+Note that you can't upload files or folders to a media collection. You have to upload the asset to the media library and add it to the media collections.
+
+{% hint style="info" %}
+**Limit**
+You can only add 100 assets to 100 media collections at once.
+{% endhint %}
+
+## Remove asset from media collections
+
+A restricted user only needs to have "manage" permission on a media collection to be able to remove assets from it. There are two ways to remove an asset from media collections-
+
+1. **Remove multiple assets from media collection**
+   Navigate into the media collection from where you want to remove assets. Select and right-click to open the dropdown, then click on "Remove from media collection." A confirmation popup would open. Click "Submit" to remove the selected assets from the media collection.
+
+2. **Remove asset from multiple media media collections**
+   Select and right-click on the asset from which you want to remove media collection associations. You can view the list of media collections that the asset was added to and for which you have "manage" permission. Select the media collections from which you want to remove the assets and click "Submit."
+
+The access and permission levels of a folder cascade to its subfolders and files. If you remove an asset from a media collection, it may still be associated with media collections if its parent is added to them.
+
+{% hint style="info" %}
+**Limit**
+* You can only remove 100 assets from a media collection at once.
+* You can only remove an asset from 100 media collections at once.
```

Comment:
at once -> at a time / in a single request / per request

Reply:
done